wp-content/themes/pub/wporg-learn-2020/css/components/_workshop-discussion-events.scss

```diff
@@ -1,10 +1,6 @@
 .wporg-learn-workshop-discussion-events {
 	:last-child {
 		margin-bottom: 0;
-
-		@media only screen and (max-width: $breakpoint-tablet) {
```

Comment:
This was actually targeting every `:last-child` in `.wporg-learn-workshop-discussion-events`, not only the buttons as I think was the intention.